docs/graphql-migrations/plugins.md

```diff
@@ -0,0 +1,7 @@
+---
+id: plugins
+title: GraphQL Migrations Plugins
+sidebar_label: Plugins
+---
+
+## TODO
```

Comment:
To be done in followup PR. Do we have a use case in mind for this options? @craicoverflow

Reply:
Copy it from the forked project:

https://graphql-migrate.netlify.app/guide/plugins.html

Reply:
Okay. That could be a very good start.